test/jdk/java/awt/print/PrinterJob/ImagePrinting/AlphaPrintingOffsets.java

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright (c) 2007, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
I guess you copied the header including "2007, 2022," from some other file.
This is a new test so it should just be "2024,"